apps/namadillo/scripts/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
Another question, what would be the benefit of using a custom script instead of actions/upload-artifact, like in the wallet.yml ci?

Reply:
There are more steps in this script that we couldn't do with actions/upload-artifact as far as I know, such as appending the build information to the README or possibly renaming the README-dist file to README.